src/cryptonote_core/cryptonote_core.h

```diff
@@ -1156,6 +1156,10 @@ namespace cryptonote
      std::unordered_map<crypto::x25519_public_key, lokimq::AuthLevel>& _lmq_auth_level_map() { return m_lmq_auth; }
      lokimq::TaggedThreadID const &pulse_thread_id() const { return *m_pulse_thread_id; }
 
+     /// Service Node's storage server and lokinet version
+     std::array<int, 3> ss_version;
+     std::array<int, 3> lokinet_version;
```

Comment:
I *think* these get passed in through the RPC layer as `uint16_t`s rather than `int`s